code/game/objects/structures/door_assembly.dm

```diff
@@ -165,143 +170,188 @@
+				user.visible_message("<b>[user]</b> starts welding the glass panel out of the airlock assembly.", SPAN_NOTICE("You start welding the glass panel out of the airlock assembly."))
+				if(do_after(user, 40 / W.toolspeed))
+					if(!src || !WT.isOn())
+						return
+					to_chat(user, SPAN_NOTICE("You weld the glass panel out."))
 					new /obj/item/stack/material/glass/reinforced(src.loc)
-					glass = 0
+					glass = FALSE
 			else if(!anchored)
-				user.visible_message("[user] dissassembles the airlock assembly.", "You start to dissassemble the airlock assembly.")
-				if(do_after(user, 40/W.toolspeed))
-					if(!src || !WT.isOn()) return
-					to_chat(user, "<span class='notice'>You dissasembled the airlock assembly!</span>")
+				user.visible_message("<b>[user]</b> starts disassembling the airlock assembly.", SPAN_NOTICE("You start disassembling the airlock assembly."))
+				if(do_after(user, 40 / W.toolspeed))
+					if(!src || !WT.isOn())
+						return
+					to_chat(user, SPAN_NOTICE("You disassemble the airlock assembly."))
 					dismantle()
 		else
-			to_chat(user, "<span class='notice'>You need more welding fuel.</span>")
+			to_chat(user, SPAN_WARNING("You need more welding fuel."))
+			return
+
+	else if(W.iswrench())
+		if(state != STATE_UNWIRED)
+			to_chat(user, SPAN_WARNING("\The [src] can only be wrenched once it's in an unwired state."))
 			return
 
-	else if(W.iswrench() && state == 0)
 		playsound(src.loc, W.usesound, 100, 1)
+	
 		if(anchored)
 			user.visible_message("<b>[user]</b> begins unsecuring the airlock assembly from the floor.", \
 								SPAN_NOTICE("You start unsecuring the airlock assembly from the floor."))
 		else
 			user.visible_message("<b>[user]</b> begins securing the airlock assembly to the floor.", \
 								SPAN_NOTICE("You start securing the airlock assembly to the floor."))
 
-		if(do_after(user, 40/W.toolspeed))
-			if(!src) return
-			to_chat(user, "<span class='notice'>You [anchored? "un" : ""]secured the airlock assembly!</span>")
+		if(do_after(user, 40 / W.toolspeed))
+			if(!src)
+				return
+			to_chat(user, SPAN_NOTICE("You [anchored? "un" : ""]secure \the [src]."))
 			anchored = !anchored
 
-	else if(W.iscoil() && state == 0 && anchored)
+	else if(W.iscoil())
+		if(state > STATE_UNWIRED)
+			to_chat(user, SPAN_WARNING("\The [src] has already been wired."))
+			return
+		if(!anchored)
+			to_chat(user, SPAN_WARNING("\The [src] must be anchored before it can be wired."))
+			return
 		var/obj/item/stack/cable_coil/C = W
 		if (C.get_amount() < 1)
-			to_chat(user, "<span class='warning'>You need one length of coil to wire the airlock assembly.</span>")
+			to_chat(user, SPAN_WARNING("You need one length of coil to wire the airlock assembly."))
+			return
+		user.visible_message("<b>[user]</b> starts wiring the airlock assembly.", SPAN_NOTICE("You start wiring the airlock assembly."))
+		if(do_after(user, 40) && state == STATE_UNWIRED && anchored)
+			if(C.use(1))
```

Comment:
Looking at this I'm thinking if it doesn't require too few. It's pretty inconsequential but still... The airlocks have quite a few wires and yet they require almost nothing compared to other machinery?